.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,20 @@
-<!--
-Thank you for your interest in and contributing to Kibana! There
-are a few simple things to check before submitting your pull request
-that can help with the review process. You should delete these items
-from your submission, but they are here to help bring them to your
-attention.
-
-- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
-- Have you followed the [contributor guidelines](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md)?
-- If submitting code, have you included unit tests that cover the changes?
-- If submitting code, have you tested and built your code locally prior to submission with `yarn test && yarn build`?
-- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
--->
\ No newline at end of file
+### Summary
+
+Summarize your PR. If it includes design elements include a screenshot or gif.
+
+### Checklist
+
+Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.
+
+- [ ] This was checked for cross-browser compatability, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
+- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing) and uses sentence case text
```

Comment:
Probably want to add something about i18n here too.  We don't want to be creating more rework for ourselves going forward.

Reply:
That's a good one. Do we have any docs on how that system works? I didn't see anything in contributing.md

Reply:
Not sure.  I would ask the folks in kibana-localization.